TgeeT/Questioner
# #163206727 meetup validation bug fix

#### What does this PR do?

This PR is meant to fix the endpoint to ensure that data is validated before being saved and return descriptive response messages
#### Description of Task to be completed?

Write validation for empty data and wrong data
Return response messages as in the requirements sheet

#### How should this be manually tested?
How should this be manually tested?

follow the steps below:
1. git clone https://github.com/tgeet/Questioner.git
2. git checkout to bg-meetup-validation-edgecase-163206727 branch
3. flask run the app and use postman to verify by trying to leave some fields empty while posting

### What are the relevant pivotal tracker stories?
#163206727

### Screenshots
##### with a relevant response spec
![image](https://user-images.githubusercontent.com/45405036/51158251-b73d8700-1894-11e9-828b-459f5de0fe9f.png)
##### with a missing field
![image](https://user-images.githubusercontent.com/45405036/51158314-f7046e80-1894-11e9-8175-f1617b5379df.png)
